frontend/console/src/features/modules/schema/LinkTokens.tsx

```diff
@@ -2,24 +2,24 @@ import { useParams } from 'react-router-dom'
 import { DeclLink } from '../decls/DeclLink'
 import { UnderlyingType } from './UnderlyingType'
 
-export const LinkToken = ({ token }: { token: string }) => {
+export const LinkToken = ({ token, containerRect }: { token: string; containerRect?: DOMRect }) => {
   const { moduleName } = useParams()
   if (token.match(/^\w+$/)) {
     return (
       <span className='font-bold'>
-        <DeclLink slim moduleName={moduleName} declName={token} />
+        <DeclLink slim moduleName={moduleName} declName={token} containerRect={containerRect} />
       </span>
     )
   }
   return token
 }
 
-export const LinkVerbNameToken = ({ token }: { token: string }) => {
+export const LinkVerbNameToken = ({ token, containerRect }: { token: string; containerRect?: DOMRect }) => {
   const splitToken = token.split('(')
   return (
     <span>
-      <LinkToken token={splitToken[0]} />
-      (<UnderlyingType token={splitToken[1]} />
+      <LinkToken token={splitToken[0]} containerRect={containerRect} />
+      (<UnderlyingType token={splitToken[1]} containerRect={containerRect} />
```

Comment:
do we need to check for `splitToken.length` before these?

Reply:
Added. It _should_ never happen, but if we add a new caller that doesn't check for it, then yeah the guard would definitely come in handy